README.md

```diff
@@ -19,8 +19,7 @@ go get -u github.com/lytics/lytics
 
 ### Usage
 
-All examples use JQ https://stedolan.github.io/jq/ to 
-prettify the json output.
+All examples use JQ https://stedolan.github.io/jq/ to prettify the json output.
 
```

Comment:
I suspect the read me needs a bit more of an update. Particularly in the example commands. For example I am seeing `segmentscan` in the README, and `segmentScan` in the new code.